packages/visual-editor/src/components/Draggable/DraggableComponent.tsx

```diff
@@ -65,15 +65,15 @@ export const DraggableComponent = ({
   }, [coordinates]);
 
   return (
-    <div ref={draggableRef}>
+    <div className={classNames('Draggable', className)} ref={draggableRef}>
```

Comment:
In #285 I actually changed up this logic a bit so that we could get rid of this wrapping div that was causing layout issues in editor mode.

Since this `div` was only introduced for the Tooltip coordinates, I opted to add a data attribute to the `div` on line 76 here which can be used to fetch and calculate the same logic that this ref was doing before.

We may want to compare this solution to this PR to see if it would work for this scenario too

Reply:
Sounds good, PR is merged so you can move forward with this one!

Reply:
Updated & reconciled, everything works great w/ tooltip placement and selected component outlines